pkg/beaconfqdn/analyzer.go

```diff
@@ -242,7 +242,7 @@ func (a *analyzer) start() {
 					"score":              score,
 					"cid":                a.chunk,
 					"src_network_name":   entry.Src.SrcNetworkName,
-					"resolved_ips":       entry.ResolvedIPs,
+					"resolved_ips":       entry.ResolvedIPs.Items(),
```

Comment:
Note that .Items() must be called to convert the map into an array. This is likely to create a merge conflict with #676 as `ResolvedIPs` is migrated from `data.UniqueIPSet` to `[]data.UniqueIP` in that PR.

https://github.com/activecm/rita/commit/cd8dd38af38d30b06b0141d08b3370435ef2ba4e
https://github.com/activecm/rita/commit/180de847eb6ddb15e008891d9d690427ef307b45